src/libraries/System.IO.Compression.Zstandard/ref/System.IO.Compression.Zstandard.cs

```diff
@@ -0,0 +1,105 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// ------------------------------------------------------------------------------
+// Changes to this file must follow the https://aka.ms/api-review process.
+// ------------------------------------------------------------------------------
+
+namespace System.IO.Compression
+{
+    public sealed partial class ZstandardCompressionOptions
+    {
+        public ZstandardCompressionOptions() { }
+        public bool AppendChecksum { get { throw null; } set { } }
+        public static int DefaultQuality { get { throw null; } }
+        public static int DefaultWindow { get { throw null; } }
+        public System.IO.Compression.ZstandardDictionary? Dictionary { get { throw null; } set { } }
+        public bool EnableLongDistanceMatching { get { throw null; } set { } }
+        public static int MaxQuality { get { throw null; } }
+        public static int MaxWindow { get { throw null; } }
+        public static int MinQuality { get { throw null; } }
+        public static int MinWindow { get { throw null; } }
+        public int Quality { get { throw null; } set { } }
+        public int TargetBlockSize { get { throw null; } set { } }
+        public int Window { get { throw null; } set { } }
+    }
+    public partial class ZstandardDecoder : System.IDisposable
```

Comment:
`ZstandardDecoder` and `ZstandardEncoder` look like they should be `sealed` - was this just an API review oversight when switching these to class?

Reply:
Yeah, just oversight.  Go ahead and seal them.